src/lib/Chip/Chip.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+  import type { Variants } from './types'
+
+  export let variant: keyof typeof Variants
```

Comment:
If `Variants` it's defined as an union type, then this could be

```suggestion
  export let variant: Variants;
```